recipe_scrapers/americastestkitchen.py

```diff
@@ -50,27 +50,30 @@ def ingredient_groups(self):
             )
         return ingredient_groups
 
-    def instructions(self):  # add headnote
+    def instructions(self):
         if headnote := self._get_additional_details.get("headnote"):
             # We could import HTMLTagStripperPlugin, but that would make it -- an optional plugin -- a dependency.
-            headnote = f"Note: {normalize_string(re.sub(r'<.*?>', '', headnote))}\n"
+            headnote = f"Note: {self._strip_html(headnote)}"
         else:
             headnote = ""
-        return headnote + self.schema.instructions()
```

Comment:
> Some recipes are generate exceptions with default `scraper.instructions()` changing to retrieving data from `_get_additional_details()`

:+1: - a test HTML file to provide coverage for this would be nice, I can probably approve without it though

Reply:
I have one, but wasn't sure how to add a 2nd test.

Reply:
@smilerz , to add a secondary test file you can just created a seperate .testhtml and associated json in the same parent folder named `americanstestkitchen_2.testhtml` (and rename the original files to `*_1.*` if possible) 

A ton of other scrapers have multiple tests if you need an example check out `addapinch`

Reply:
Due to some other changes I suggested, we ended up with separate test directories (one for each of the brand name domains) -- just mentioning that in case there's any later confusion about why there aren't any multi-test JSON directories added here.